src/xpcc/driver/pressure/bmp085_data.hpp

```diff
@@ -0,0 +1,191 @@
+// coding: utf-8
+/* Copyright (c) 2016, Roboterclub Aachen e. V.
+ * All Rights Reserved.
+ *
+ * The file is part of the xpcc library and is released under the 3-clause BSD
+ * license. See the file `LICENSE` for the full license governing this code.
+ */
+// ----------------------------------------------------------------------------
+
+#include <stdio.h>
+#include <cmath>
+
+#include <xpcc/debug/logger/logger.hpp>
+
+#ifndef XPCC_BMP085_DATA_HPP
+#define XPCC_BMP085_DATA_HPP
+
+#undef  XPCC_LOG_LEVEL
+#define XPCC_LOG_LEVEL xpcc::log::DISABLED
+
+// Forward declaration the test class
+class Bmp085Test;
+
+namespace xpcc
+{
+
+template < typename I2cMaster >
+class Bmp085;
+
+namespace bmp085data
```

Comment:
I see. So `xpcc::bmp085data` is a `detail` namespace. Is the user ever expected to interact with it? No, right?

Your example uses `xpcc::bmp085data::Calibration`, perhaps you could alias that into the `bmp085` struct?
```cpp
using Calibration = xpcc::bmp085data::Calibration;
```